cmd/kube-scheduler/app/server.go

```diff
@@ -293,7 +293,7 @@ func buildHandlerChain(handler http.Handler, authn authenticator.Request, authz
 
 func installMetricHandler(pathRecorderMux *mux.PathRecorderMux) {
 	configz.InstallHandler(pathRecorderMux)
-	defaultMetricsHandler := legacyregistry.Handler().ServeHTTP
+	defaultMetricsHandler := promhttp.Handler().ServeHTTP
```

Comment:
@logicalhan does this change look right to you?

Reply:
No. This is going to break our metrics. We should just add the ignore lint directive above the line for the time being.

Reply:
Done, added as:
```go
//lint:ignore SA1019 See the Metrics Stability Migration KEP
```